lib/adapter.js

```diff
@@ -24,9 +24,12 @@ const DEFAULT_OPTS = {
     profile: [], // <string> (name) specify the profile to use
     require: [], // <string> (file/dir) require files before executing features
     snippetSyntax: undefined, // <string> specify a custom snippet syntax
-    strict: false, // fail if there are any undefined or pending steps
+    strict: false, // <boolean> fail if there are any undefined or pending steps
     tags: [], // <string[]> (expression) only execute the features or scenarios with tags matching the expression
-    timeout: DEFAULT_TIMEOUT // timeout for step definitions
+    timeout: DEFAULT_TIMEOUT, // <number> timeout for step definitions in milliseconds
+    ignoreUndefinedDefinitions: false, // <boolean> Enable this config to treat undefined definitions as warnings.
+    failAmbiguousDefinitions: false // <boolean> Enable this config to treat ambiguous definitions as errors.
+
```

Comment:
Are these two options part of the cucumber framework? If not we should remove them from here to avoid side effects.